templates/internet-of-things/management.html

```diff
@@ -10,343 +10,493 @@
   Securely and reliable deploy, manage and scale your embedded Ubuntu devices. Get low-touch and custom workflows for any solution.
 {% endblock %}
 
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+{% from "_macros/vf_quote-wrapper.jinja" import vf_quote_wrapper %}
+{% from "_macros/vf_tiered-list.jinja" import vf_tiered_list %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
 {% block outer_content %}
```

Comment:
what's the role of this `outer_content` block?

Reply:
Was defined as a placeholder in base template file, and extended in the current file. But given the fact it doesn't really provide anything on its own rather than nesting inner content, it could be removed all together. Though, I left it as it is for better readability.